api/swagger/swaggerDoc.yaml

```diff
@@ -802,6 +1044,22 @@ definitions:
       detail:
         type: object
         example: {}
+  UsernameResponseV03:
+    type: object
+    properties:
+      data:
+        $ref: '#/definitions/UsernameResponseModelV03'
+  UsernameResponseModelV03:
+    properties:
+      type:
```

Comment:
Is there anything we can call this other than "type"?  It seems confusing given that the data type is also "type"...and then there's something else called "cardType"...

Reply:
It's the standard for good and error messages though. But I see the confusion with `cardType`. The other `type` is part of Swagger but yes, it's a bit confusing.